chapters/classes.tex

```diff
@@ -164,6 +164,7 @@ \subsection{Static Semantics}\label{component-declaration-static-semantics}
 % Seems sufficient to just have \indexinline variant of 'partial' in index.
 A class defined with \lstinline!partial!\indexinline{partial} in the \lstinline[language=grammar]!class-prefixes! is called a \firstuse[---]{partial} class.
 Such a class is allowed to be incomplete, and cannot be instantiated in a simulation model; useful, e.g., as a base class.
+See \cref{short-class-definitions} regarding short class definition semantics of \lstinline!partial!.
```

Comment:
Since the section is about a page long it would be better to have a more detailed reference (so that at least people who know it can see it as a reminder), e.g., 
```suggestion
See \cref{short-class-definitions} regarding short class definition semantics of propagating \lstinline!partial!.
```
At first I considered "inherting" - with quotes, but instead of saying that it "kind of inherits" partial, we can say that it actually propagates "partial".

Reply:
Sure.